### PR TITLE
Replace 'master' with 'main'

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Is there something we need to configure to ensure the necessary drop share channels (not darc channels) are configured? I'm not sure if it's completely controlled by the settings in this repo.

### CODE REVIEW

**Review by wtgodbe — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Contributing
 ======
 
-Information on contributing to this repo is in the [Contributing Guide](https://github.com/aspnet/AspNetCore/blob/master/CONTRIBUTING.md) in the AspNetCore repo.
+Information on contributing to this repo is in the [Contributing Guide](https://github.com/dotnet/AspNetCore/blob/main/CONTRIBUTING.md) in the AspNetCore repo.
```
Comment (dougbu) [inline CONTRIBUTING.md:4]: nit
```suggestion
Information on contributing to this repo is in the [Contributing Guide](https://github.com/dotnet/aspnetcore/blob/main/CONTRIBUTING.md) in the AspNetCore repo.
```

File: build/dependencies.props [hunk 2]
```diff
@@ -19,7 +19,7 @@
   </PropertyGroup>
 
   <!--
-    Attempt to keep these mostly aligned with https://github.com/dotnet/toolset/blob/master/env/Versions.props
+    Attempt to keep these mostly aligned with https://github.com/dotnet/toolset/blob/main/env/Versions.props
```
Comment (dougbu) [inline build/dependencies.props:22]:
```suggestion
    Attempt to keep these mostly aligned with https://github.com/dotnet/toolset/blob/master/eng/Versions.props
```
No 'main' branch in this repo. It still has a 'master' branch and a nearly-empty 'archive' branch for some reason.

**Review by dougbu — APPROVED** [review]

**Comment by dougbu** [comment]
> Is there something we need to configure to ensure the necessary drop share channels (not darc channels) are configured?

As far as I know, nothing should be using the 'master' channel. Our 2.1 build still uses the 2.1 channel but that may be about it.

**Comment by dougbu** [comment]
/fyi release pipeline at https://dev.azure.com/dnceng/internal/_releaseDefinition?definitionId=1 should automatically create the expected channel URIs

**Pull request merged**